home.css

```diff
@@ -0,0 +1,250 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400&family=Roboto+Slab:wght@300&display=swap');
+*{
+    margin:0;
+    padding: 0;
+    box-sizing: border-box;
+ }
+ a{
+     text-decoration: inherit;
+     color: inherit;
+ }
+ li{
+    list-style: none;
+ }
+ :root{
+     --color-navbar:#f5f5f5;
```

Comment:
Suggestion: for readability perspective: you use one space after : 
example: `color: #fff` instead of `color:#fff`
also `li {` instead of `li{`

Reply:
Sure, will do the suggested changes @Meghana-sk